### PR TITLE
updated more javascript dom calls to use jQuery instead

### PR DESCRIPTION
**Pull request by NickWilsonDev**
@chrisgoodell check this out testing

### CODE REVIEW

**Comment by NickWilsonDev** [comment]
@chrisgoodell testing

File: index.html [hunk 1]
```diff
@@ -20,6 +20,9 @@ <h1>Instagram Clone</h1>
     <div id="detailed-view"></div>
     </center>
     <!-- usually call js right before end of body tag -->
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js"
```
Comment (chrisgoodell) [inline index.html:23]: this looks pretty good 👍 
***great job***

File: index.html [hunk 2]
```diff
@@ -20,6 +20,9 @@ <h1>Instagram Clone</h1>
...
+      integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
```
Comment (chrisgoodell) [inline index.html:24]: SHA256 💯

**Review by chrisgoodell — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (NickWilsonDev) [reply index.html:23]: Thanks

**Pull request merged**